content/mobile.md

```diff
@@ -0,0 +1,491 @@
+
+You install plugins from the local file system by specifying a `file://` URL, which gets interpreted relative to the project directory:
+
+```sh
+meteor add cordova:cordova-plugin-underdevelopment@file://../plugins/cordova-plugin-underdevelopment
+```
+
+Meteor packages can also depend on plugins installed from the local file system, although this probably only makes sense for local packages:
+
+```js
+Cordova.depends({
+    'cordova-plugin-underdevelopment': 'file://../plugins/cordova-plugin-underdevelopment'
+});
+```
+
+<h4>Removing directly installed plugins</h4>
+
+You can remove a previously added plugin using `meteor remove`:
+
+```sh
+meteor remove cordova:cordova-plugin-camera
+meteor remove cordova:com.phonegap.plugins.facebookconnect
+meteor remove cordova:cordova-plugin-underdevelopment
+```
+
+<h3 id="using-plugins">Using plugins</h3>
+
+You should wrap any functionality which relies on a Cordova plugin in a `Meteor.startup()` block to make sure the plugin has been fully initialized (by listening to the `deviceready` event). For example, when using the Cordova geolocation plugin:
+
+```js
+// The plugin may not have been initialized here
+navigator.geolocation.getCurrentPosition(success);
+
+Meteor.startup(function() {
+    // Here we can be sure the plugin has been initialized
+    navigator.geolocation.getCurrentPosition(success);
+});
+```
+
+<h3 id="cordova-specific-javascript">Adding Cordova-specific Javascript code to your application</h3>
+
+Just as you can use `Meteor.isServer` and `Meteor.isClient` to separate your client-side and server-side code, you can use `Meteor.isCordova` to separate your Cordova-specific code from the rest of your code.
+
+```js
+if (Meteor.isServer) {
+  console.log("Printed on the server");
+}
+
+if (Meteor.isClient) {
+  console.log("Printed in browsers and mobile apps");
+}
+
+if (Meteor.isCordova) {
+  console.log("Printed only in mobile Cordova apps");
+}
+```
+
+In addition, packages can include a different set of files for Cordova builds and browser builds with `addFiles`:
+
+- `api.addFiles('foo.js', 'web.cordova')`: includes `foo.js` in only Cordova builds.
+- `api.addFiles('bar.js', 'web.browser')`: includes `bar.js` in only browser builds.
+- `api.addFiles('baz.js', 'web')`: includes `baz.js` in all client builds.
+
+The same syntax can be used for `api.use`, `api.imply`, and `api.export`.
+
+<h2 id="accessing-local-files-and-remote-resources">Accessing local files and remote resources</h2>
+
+<h3 id="accessing-local-files">Accessing local files</h3>
+
+[Most of this section still needs to be written.]
```

Comment:
Instead of "this needs to be written" can we just link to external content for these sections?